docs/book/src/user/concepts.md

```diff
@@ -13,7 +13,7 @@ A Kubernetes cluster whose lifecycle is managed by a Management Cluster.
 
 ### Infrastructure provider
 
-A source of computational resources, such as Machines and networking. For example, cloud Infrastructure Providers include AWS, Azure, and Google, and bare metal Infrastructure Providers include VMware, MAAS, and metal3.io.
+A source of computational resources, such as compute and networking. For example, cloud Infrastructure Providers include AWS, Azure, and Google, and bare metal Infrastructure Providers include VMware, MAAS, and metal3.io.
```

Comment:
```suggestion
A source of computational resources, such as virtual machines and networking. For example, cloud Infrastructure Providers include AWS, Azure, and Google, and bare metal Infrastructure Providers include VMware, MAAS, and metal3.io.
```
Personally I think that `compute` is a little bit too generic for an example (not blocking)

Reply:
I suppose technically we could have a provider that interacts with baremetal machines.

Maybe we just go back to `machines` -- honestly reading `Machines` just came off a bit weird since we refer to it throughout the docs when talking about the `Machine` CRD.

Reply:
Agree with @joshrosso in this case. In context, "compute" should be understood ok and covers both virtual and bare metal cleanly.

Reply:
kk